Makefile

```diff
@@ -0,0 +1,69 @@
+.PHONY: build-garage-headless build-garage-nvidia \
+	run-garage-headless run-garage-nvidia
+
+# Path in host where the experiment data obtained in the container is stored
```

Comment:
isn't this set in config.py/config_personal.py?

there needs to be a single source of truth here. you might need to ask Python where to put it.

Reply:
Done

Reply:
that solution (attempting to glob the conda install) is really hacky and makes me nervous. you have no guarantee that conda is even inside $HOME. i have machines where it lives in /opt, for instance. furthermore, you have no guarantee the host is running bash.

this is a good indication that at least one of the following things are true:
* Makefile is not a good way for us to manage the Dockerfiles (current config assumes Docker is launched from Python, so doesn't have this problem)
* We should not configure the user's environment in config.py

I think the least worst solution here is to pick a default place to map data/ from the docker container into (perhaps `$(pwd)/data/dockerfile_name` and stick to it. Allow users to override the default with an argument  to the Makefile

It also means we should remove the Docker-related entries from config/config_personal.py

It's clear here that how we do configuration in garage needs to be rewritten, but that can be left for another day.

Reply:
For headless and nvidia I'm doing the following binding:
```
-v $(DATA_PATH)/$(CONTAINER_NAME):/root/code/garage/data
```
Where `DATA_PATH` is equal to `$(shell pwd)/data`
By default, `CONTAINER_NAME` for headless is equal to `garage-headless-$(DATE_TIME)`, and `garage-nvidia-$(DATE_TIME)` for nvidia. The DATE_TIME is defined by `$(shell date "+%Y_%m_%d_%H_%M_%S")`.
`CONTAINER_NAME` can be set by the user by passing it as an argument when calling make.

I created the target copy_config_personal to copy the config_personal.py file in case it doesn't exist.

Reply:
i think the path should be `$(shell pwd)/data/$(CONTAINER_NAME)`. the default experiment runner config already adds timecodes.